documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-management-tasks.adoc
+
+[id='proc-setting-broker-limits-{context}']
+
+= Setting limits on brokers using the Kafka Static Quota plugin
+
+[role="_abstract"]
+Use the _Kafka Static Quota_ plugin to set throughput and storage limits on brokers in your Kafka cluster.
+You can set a byte-rate threshold and storage quotas for clients interacting with your brokers.
+You enable the plugin and set limits by configuring the `Kafka` resource.
+
+You can set byte-rate thresholds for producer and consumer bandwidth.
+The total limit is distributed across all clients accessing the broker.
+For example, you can set a byte-rate threshold of 40 MBps for producers.
+If you have two producers running, they are each limited to a throughput of 20 MBps.
+
+Storage quotas throttle Kafka disk storage limits between a soft limit and hard limit.
```

Comment:
I guess we should explain how JBOD storage is treated - as a sum of all used space. So for example when users has broker with 2 1TB disks and the quota is set 1,1TB, it can easily allow situation when one disk gets completely full and other is almost empty.

Reply:
Added as a note

> NOTE: For JBOD storage, the limit applies across all disks. If a broker is using two 1 TB disks and the quota is 1.1 TB, one disk might fill and the other disk will be almost empty.